addons/calendar/models/calendar.py

```diff
@@ -1809,7 +1809,7 @@ def _sync_activities(self, values):
             if values.get('name'):
                 activity_values['summary'] = values['name']
             if values.get('description'):
-                activity_values['note'] = values['description']
+                activity_values['note'] = tools.plaintext2html(values['description'] or '')
```

Comment:
from line above we know that `values.get('description')` is not falsy so `or ''` should never happen